.circleci/config.yml

```diff
@@ -145,43 +79,17 @@ workflows:
             # run for all branches and tags
             tags:
               only: /.*/
-      - build_base_docker:
-          requires:
-            - checkout
-          filters:
-            # run for all branches and tags
-            tags:
-              only: /.*/
       - lint:
```

Comment:
You can have as many jobs as you want, but the main, most useful ones are:
- `test` which runs the unit tests
- `lint` which runs our linter

Also note how both of these jobs require `checkout` to make the code available. The idea is that by default, circle will run all jobs in parallel unless otherwise noted. By having a special `checkout` task that downloads the code from github and adding the `requires` statement on the other jobs, we can only download the code once, but do linting and testing in parallel which lets us save time!